llvm/lib/Target/NVPTX/NVPTXUtilities.cpp

```diff
@@ -179,6 +179,13 @@ static bool argHasNVVMAnnotation(const Value &Val,
   return false;
 }
 
+static std::optional<unsigned> getFnAttrParsedIntOrNull(const Function &F,
```

Comment:
Nit: `OrNull` is kind of implied by return type being optional.

Reply:
Removed